Interop/Indexer/build.gradle

```diff
@@ -76,7 +76,7 @@ model {
         clangstubs(NativeLibrarySpec) {
             sources {
                 c.source.srcDir 'prebuilt/nativeInteropStubs/c'
-		        cpp.source.srcDir 'prebuilt/nativeInteropStubs/cpp'
+                cpp.source.srcDir 'prebuilt/nativeInteropStubs/cpp'
```

Comment:
`signalChaining.cpp` is not a prebuilt stub, so it should be located in other directory.

Reply:
Which one?

Reply:
E.g. `src/nativeInteropStubs/cpp`

Reply:
BTW, seems `disable-abi-checks.cpp` is also not generated.